spec/classes/rabbitmq_spec.rb

```diff
@@ -282,9 +284,9 @@
   end
 
   context 'on Redhat' do
-    let(:params) {{ :manage_repos => true }}
+    let(:params) {{ :manage_repos => true, :repos_ensure => true }}
     let(:facts) {{ :osfamily => 'RedHat', :operatingsystemmajrelease => '7' }}
-    it 'does import repo public key when manage_repos is true' do
+    it 'does import repo public key when manage_repos and repos_ensure are true' do
```

Comment:
Please note this change.  If you disagree with it, then it may be needed to look again at the logic in init.pp, as that checks for manage_repos != false as the condition to include repo class.  Since this parameter is itself deprecated, it seems to me this logic should change, though it is not a requirement for this patch.